plugins/lookup/hashi_vault.py

```diff
@@ -42,22 +42,25 @@
     token_path:
       description: If no token is specified, will try to read the token file from this path.
       env:
-        - name: VAULT_TOKEN_PATH
+        - name: ANSIBLE_HASHI_VAULT_TOKEN_PATH
+          version_added: '1.0.0'
```

Comment:
(Maybe I already asked that before, but...) Is there a reason for not keeping the old names (and deprecating them)? That way, not every use of the plugin has to be checked and changed immediately, but people have a version where they get deprecation warnings and have time to remove usage of all "wrong" variables, until they are effectively removed.

Reply:
My thinking was this is  a new major version, so might as well go for removal. 
But for these 3 that are being removed, I'd be open to deprecation. Is it supported with an additional attribute? Like I can I have 2 env vars in the list where one of them is deprecated by adding a `deprecated:` field or something? I kinda remember there being something like that (a lot of chatter about how to support it in collections, whether it should be version or time-based, etc.) but I can't seem to find the docs.

Reply:
Do you think it's too fast to add the new options in 0.2.0 and remove the old in 1.0.0? It's short in calendar time but it does allow for a version where both work.

Reply:
For end-users (who don't explicitly use the new collection before there will be a redirect in community.general 2.0.0), it will look like it worked in Ansible 2.10.x, and suddenly stopped working in Ansible 2.11. It should at least be deprecated for some time. There should be a proper deprecation period for Ansible users.